tools/release/sync-stable-blocks.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-console */
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+
+/**
+ * Constants
+ */
+const BLOCK_LIBRARY_SRC = 'node_modules/@wordpress/block-library/src/';
+
+const REPOSITORY_ROOT = path.dirname(path.dirname(__dirname));
+
+/**
+ * The main function of this task.
+ *
+ * Refreshes the PHP files referring to stable @wordpress/block-library blocks.
+ */
+function main() {
+	const blocks = getStableBlocksMetadata();
+
+	// wp-includes/blocks/require-blocks.php
+	console.log( 'Refreshing wp-includes/blocks/require-blocks.php...' );
+	const PHPRequires = blocks
+		.filter( isDynamic )
+		.map( toDirectoryName )
+		// To PHP require statement:
+		.map( dirname => `require ABSPATH . WPINC . '/blocks/${ dirname }.php';` )
+		.join( "\n" );
+
+	fs.writeFileSync(
+		`${ REPOSITORY_ROOT }/src/wp-includes/blocks/require-blocks.php`,
+		`<?php \n\n
+// This file was autogenerated by tools/release/sync-stable-blocks.js, do not change manually!
+// Include files required for core blocks registration.
+${ PHPRequires }
+`,
+	);
+
+	// tests/phpunit/includes/unregister-blocks-hooks.php
+	console.log( 'Refreshing tests/phpunit/includes/unregister-blocks-hooks.php...' );
+	const unregisterHooks = blocks.filter( isDynamic )
+		.map( function toHookName( metadata ) {
+			const php = fs.readFileSync( path.join( metadata.path, '..', 'index.php' ) ).toString();
+			let hookName = php.substr( php.indexOf( "add_action( 'init', 'register_block_core_" ) );
+			return hookName.split( "'" )[ 3 ];
+		} )
+		.map( function toUnregisterCall( hookName ) {
+			return `remove_action( 'init', '${ hookName }' );`;
+		} )
+		.join( "\n" );
+
+	fs.writeFileSync(
+		`${ REPOSITORY_ROOT }/tests/phpunit/includes/unregister-blocks-hooks.php`,
+		`<?php
+
+// This file was autogenerated by tools/release/sync-stable-blocks.js, do not change manually!
```

Comment:
Not that important: `tools/release/sync-stable-blocks.js` - should we put that inside the constant so it is dynamically computed from `__filename`?